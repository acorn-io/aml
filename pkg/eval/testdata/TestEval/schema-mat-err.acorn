schema {
    a: string =~ "hi.*"
    a: string !~ "bye.*"
}

a: "bye"
