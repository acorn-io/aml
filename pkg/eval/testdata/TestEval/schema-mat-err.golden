"schema violation .a: unmatched constraint bye =~ hi.* (schema-mat-err.acorn:1:8)"
